packages/manager/overthebox/rollup.config.js

```diff
@@ -4,14 +4,21 @@ const config = rollupConfig({
   input: 'src/index.js',
 });
 
-export default [
-  config.cjs(),
-  config.umd({
-    output: {
-      globals: {
-        angular: 'angular',
-        telecomUniverseComponents: '@ovh-ux/telecom-universe-components',
+const outputs = [];
```

Comment:
```suggestion
const outputs = [config.cjs()];
```
In that way you can get rid of L9.